SHAPE_DUPLICATION_DEBUG.md

```
@@ -0,0 +1,204 @@
+# Shape Duplication Bug - Debug & Test Plan
+
+## Problem
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)